docs/First-Installation.md

```diff
@@ -58,24 +58,24 @@ The simplest way to get most of these components is through an all-in-one kit su
    * For macOS: [Raspberry Pi Imager](https://www.raspberrypi.org/software/), [Etcher](https://www.balena.io/etcher/), [Apple Pi Baker](https://www.tweaking4all.com/hardware/raspberry-pi/applepi-baker-v2/), or the `dd` command
    * For Linux: [Raspberry Pi Imager](https://www.raspberrypi.org/software/), [Etcher](https://www.balena.io/etcher/), or the `dd` command
     * MacOS/Linux users can optionally extract the .img image from the downloaded .img.gz by using `gunzip` (macOS users can also simply double-click it)
-4.  Once the program's image/OS and SD card have been chosen, *write* it to the SD card and wait until the operation completes
-5.  Remove the MicroSD card from your computer, slide it back into the slot on your Raspberry Pi, and turn it on
+4. Once the program's image/OS and SD card have been chosen, *write* it to the SD card and wait until the operation completes
+5. Remove the MicroSD card from your computer, slide it back into the slot on your Raspberry Pi, and turn it on
 
 ## Configure Controllers
 
-On first boot your filesystem will be expanded automatically, you will then be welcomed with the following screen- this menu will configure your controls for both EmulationStation and RetroArch Emulators:
+On first boot you will be welcomed with the screen below. This menu will configure your controls for both EmulationStation and all RetroArch emulators:
 
 ![welcomescreen](https://cloud.githubusercontent.com/assets/10035308/9140482/cf42f25c-3cee-11e5-8f91-c1fc1c57175c.png)
 
-Hold down any button on your keyboard or gamepad and the name will appear at the bottom and then open up into a configuration menu:
+Hold down any button on your controller or keyboard. While holding, its name will appear at the bottom of the window for a few seconds and then open up into a configuration menu:
 
-![welcomescreengamepadname](https://cloud.githubusercontent.com/assets/10035308/9140505/f5c19e38-3cee-11e5-965e-0e4e85ddaf56.png)
+![welcomescreengamepadconfigure](https://cloud.githubusercontent.com/assets/10035308/9140518/0263b9c8-3cef-11e5-922f-42f790f3be91.png)
 
-Follow the onscreen instructions to configure your gamepad- if you run out of buttons just hold down a button to skip each unused button. **When you get to OK press the button you have configured as "A"**.
+Follow the onscreen instructions to configure your gamepad. If your controller doesn't have a button that you're being asked to define, just hold down any button to skip it.
 
-![welcomescreengamepadconfigure](https://cloud.githubusercontent.com/assets/10035308/9140518/0263b9c8-3cef-11e5-922f-42f790f3be91.png)
+The [Hotkey](First-Installation#Hotkeys) button is explained in detail below, but for now it is suggested to be defined as the **Select** button.
```

Comment:
The _Select_ button is not really _recommended_, is a fallback button chosen when there's no Hotkey button selected.
As it happens, there are a lot of controllers that don't have a spare button, so _Select_ ends up as the default Hotkey button, as suggested by EmulationStation's input configuration.

Reply:
> [It is suggested to use the Select button as the hotkey.](https://retropie.org.uk/docs/First-Installation/#hotkey)

^ The original text. Is there something you'd like me to change here?

Reply:
Maybe rephrase something along these lines:

> When the controller doesn't have enough buttons for a dedicated _Hotkey_, EmulationStation will propose to the _Select_ button instead. It's strongly recommended to accept this suggestion and have a working _Hotkey_ button.

Reply:
Combined with the original section? So something like this?

> The Hotkey Enable button is explained in detail below, but for now it is suggested to be defined as the Select button. When the controller doesn't have enough buttons for a dedicated Hotkey Enable button, EmulationStation will propose to use the Select button instead. It's strongly recommended to accept this suggestion to have a working Hotkey Enable button.

(Found a few more places I can make the change from Hotkey -> Hotkey Enable as requested by dankcushions [here](https://github.com/RetroPie/RetroPie-Docs/pull/63#discussion_r576679513))

Reply:
that reads a bit repeated to me, and I still don't like the suggestion "for now". I would say something like

> When the controller doesn't have enough buttons for a dedicated Hotkey Enable button, it is suggested to use the same button as Select. Hotkey functionality is explained further below.

To be honest I don't see the utility of breaking out this section and moving it below, and then needing to reference it here - just keep the table in the same place (ie, all within the 'Hotkey' section, after the controller layouts)

Reply:
> Combined with the original section? So something like this?

You can remove the original wording.

Reply:
I think in _my_ head "for now" sounded reasonable because I'm getting the impression that the Home button on most controllers is more common and more useful, especially when dealing with games that use the Select button a lot (Super Metroid). Home button being the central X on Xbox controllers, or the P on Playstation controllers, the "Logitech" button on Logitech controllers, etc. But not all controllers have one, hence the importance of including a good alternative - Select.

Reply:
Mmm I agree that a dedicated hotkey enable button is best (I use the PlayStation button), so I agree that the wording should reflect that dedicated is best, but select is a fine fallback, but I still don't see how that means you should use Select initially ("for now") and then change it later? Just use the bind the right button first time. I think my suggested wording above reflects this.

Reply:
Agreed